sgxs-tools/src/sgx_detect/tests/scaffold.rs

```diff
@@ -24,6 +24,18 @@ pub trait Name {
     fn name(&self) -> &'static str;
 }
 
+#[derive(Copy, Clone, Deserialize, Serialize, Debug, PartialEq)]
+pub enum BuildType {
```

Comment:
As previously mentioned “build” doesn't convey the right meaning. Maybe a better term is `EnvConfig` or `EnvType`